docs/support/tutorials/services-for-courses.md

```diff
@@ -0,0 +1,49 @@
+# CSC Services for Courses
+
+This guide will explain how to use the course project type with CSC's
+high-performance computing (HPC) and cloud services.
+
+## Prerequisites
+
+Staff members from higher education institutions and state research institutes can create a MyCSC course project. What you will need is:
```

Comment:
Now there is circular dependency, to create a course project you need a course project. Should be reworded.

We should move here also to info that noppe does not need a course project